systemtests/init_test.go

```diff
@@ -34,6 +34,21 @@ type systemtestSuite struct {
 	// user       string
 	// password   string
 	// nodes      []string
+	acinfo ACInfo
+}
+type ACInfo struct {
+	IP                string `json:"IP"`
+	HostIPs           string `json:"HOSTIPS"`
+	HostUsernames     string `json:"HOSTUSERNAMES"`
```

Comment:
Convention for JSON field names is to use lower case letters(not caps).
Can you please change it to lower case?